spec/mediators/translation_update_mediator_spec.rb

```diff
@@ -287,4 +287,17 @@
       expect(translation.reviewer).to be_nil
     end
   end
+
+  describe "#top_fuzzy_match" do
+    it "retrieves the top fuzzy match for a given translation" do
+      FactoryGirl.create(:translation, copy: "Testing", source_copy: 'Testing', approved: true, translated: true)
+
+      regenerate_elastic_search_indexes
+      sleep(1)
+
+      translation = FactoryGirl.build(:translation, copy: "Test", source_copy: 'Test')
+      mediator = TranslationUpdateMediator.new(translation, reviewer, ActionController::Parameters.new)
+      expect(mediator.send(:top_fuzzy_match, translation)).to eq 72.72727272727273
```

Comment:
This test needs work. I have a couple suggestions:

1. Is it possible / would it make sense to test top_fuzzy_match behavior via a test of `update_single_translation!` instead? That would be preferred since `top_fuzzy_match` is supposed to be a private method.
2. calling `sleep` inside of a test is highly undesirable. Can you mock out the response from elasticsearch instead?
3. The use of 72.72727272727273 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)) and I would like to avoid testing like this. It either needs to be explained or dropped. If we test the method via `update_single_transaction!` then we will not have this problem.